test/hotspot/jtreg/runtime/ErrorHandling/MachCodeFramesInErrorFile.java

```diff
@@ -180,6 +182,7 @@ private static void extractFrames(String hsErr, Set<String> frames, boolean nati
                 frames.add(line);
             }
         }
-        throw new RuntimeException("\"" + marker + "\" line missing in hs_err_pid file:\n" + hsErr);
+        System.err.println(hsErr);
```

Comment:
It's a popular (and reasonable) [question](https://github.com/openjdk/jdk/pull/8094#discussion_r852787701) as to why I made the change to the exception reporting. This is the error that resulted from a too-long exception message.

```
java.io.UTFDataFormatException: encoded string (`main' t....11 END.) too long: 116222 bytes
        at java.base/java.io.DataOutputStream.writeUTF(DataOutputStream.java:369)
        at java.base/java.io.DataOutputStream.writeUTF(DataOutputStream.java:333)
        at com.sun.javatest.regtest.agent.AgentServer.writeStatus(AgentServer.java:351)
        at com.sun.javatest.regtest.agent.AgentServer.doMain(AgentServer.java:307)
        at com.sun.javatest.regtest.agent.AgentServer.run(AgentServer.java:233)
        at com.sun.javatest.regtest.agent.AgentServer.main(AgentServer.java:70)
```

This error hampered my ability to debug this test. It's not very clear from the message as to where it's coming from, and it occurs sporadically depending on the length of the message produced. I eventually traced it to the line: `throw new RuntimeException("\"" + marker + "\" line missing in hs_err_pid file:\n" + hsErr);`. To trigger the error for this comment, I changed `hsErr` to `hsErr + hsErr`.

This issue may be platform specific, or the issue just isn't encountered very often because it's uncommon to add > 64KiB Strings to an Exception. Either way, making this change made the test failures much easier to diagnose.